### PR TITLE
[Aquaria] Adds Poptracker Pack to the Aquaria Setup Guides 

### PR DESCRIPTION
**Pull request by palex00**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

### CODE REVIEW

**Comment by palex00** [comment]
To explain: I tried to re-name the branch after PR-ing and I guess that counts as deletion.

**Comment by NewSoupVi** [comment]
> To explain: I tried to re-name the branch after PR-ing and I guess that counts as deletion.

Yeah, renaming a branch is not really a thing™ :D The name **is** the unique key that identifies a branch afaik

Anyway, this is fine. ~~In fact, I wonder if there just aren't that many ways to write one of these, or if some of this is copied (maybe multiple times), since it feels a bit familiar :D~~

Some peer reviews from native speakers couldn't hurt though.

**Comment by palex00** [comment]
For what it's worth this is only the English guide. Tioui, the world
maintainer, is french and will update the French guide soon through this
same branch PR. So that's why it's drafted until then

NewSoupVi ***@***.***> schrieb am Di., 8. Okt. 2024, 23:34:

**Comment by palex00** [comment]
The French Guide was translated by tioui himself

**Review by tioui — APPROVED** [review]
I just tried the procedure and it work as it should. Thanks for your work Palex00.

**Comment by OctaveC** [comment]
Hi! Sophie asked me to review the french setup guide, which has quite a few grammar errors, and overall was very literally translated with very awkward turn of phrases, so I added suggestions to fix it.

File: worlds/aquaria/docs/setup_fr.md
Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:6]: I agree with this proposition.

File: worlds/aquaria/docs/setup_fr.md
Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:134]: > Aquaria à un tracker complet qui supporte le tracking automatique.

should be:

Aquaria a un tracker complet qui supporte le tracking automatique.

> Mettre le paquet du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.

Maybe it is because of my technical skill, but I still prefer my phrasing.

> Pour activer le tracking automatique, cliquez sur le symbole "AP" en haut de la fenêtre.

I think that "en haut" is more familiar than "dans le haut".

For the rest, I don't have any objection.

File: worlds/aquaria/docs/setup_fr.md
Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:11]: It looks the same to me.

Comment (palex00) [reply worlds/aquaria/docs/setup_fr.md:12]: it's "pour" versus "à". 

You proposed pour,
Ciprocreep proposed à

Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:12]: I know. When I say it look the same, I wanted to emphases the fact that either work equally for me.

Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:12]: In my opinion it sounds better with pour (more formal)

File: worlds/aquaria/docs/setup_fr.md
Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:135]: I think it would be better to call the zip file a fichier.

File: worlds/aquaria/docs/setup_fr.md
Comment (benny-dreamly) [inline worlds/aquaria/docs/setup_fr.md:130]:
```suggestion
2. Mettre le fichier du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
```

Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:130]: Fichier sounds better than Paquet for me.

**Review by benny-dreamly — CHANGES_REQUESTED** [review]

**Comment by benny-dreamly** [comment]
Just my two cents as a French speaker from Canada.

File: worlds/aquaria/docs/setup_fr.md
Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:130]: Good for me. I did not know how to translate package.

Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:130]: Fichier is technically the word for file in French but yeah

Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:130]: I know. But it make sense.

Comment (Cipocreep) [reply worlds/aquaria/docs/setup_fr.md:130]: I dunno, feel like that might be confusing, I still feel like "dossier" is more fitting.

Of course, that's all small potatoes, it's basically fine as-is.

Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:130]: From my somewhat brief jaunt into the English French dictionary on Linguee it says that zip file is “fichier compressé”

**Comment by benny-dreamly** [comment]
The English part of the PR looks fine to me for now but I’m going to hold off on approving it until the French looks good.

File: worlds/aquaria/docs/setup_fr.md
Comment (palex00) [reply worlds/aquaria/docs/setup_fr.md:135]: Could someone suggest a complete version?

File: worlds/aquaria/docs/setup_fr.md
Comment (palex00) [reply worlds/aquaria/docs/setup_fr.md:130]: Would everyone be okay with "fichier compressé"? @Cipocreep @tioui

Comment (Cipocreep) [reply worlds/aquaria/docs/setup_fr.md:130]: I mean I guess? It might just introduce more confusion, like, the poptracker pack doesn't need to be zipped (it can be, I'm pretty sure, but it doesn't NEED to be), I feel like either "fichier" or "dossier" alone are better.

Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:130]: > Would everyone be okay with "fichier compressé"? @Cipocreep @tioui

Yes. Good form me.

File: worlds/aquaria/docs/setup_fr.md
Comment (tioui) [reply worlds/aquaria/docs/setup_fr.md:135]: > Could someone suggest a complete version?

Here is what I propose, base on the discussions:

```suggestion
# Tracking automatique

Aquaria a un tracker complet qui supporte le tracking automatique.

1. Téléchargez [Aquaria AP Tracker](https://github.com/palex00/aquaria-ap-tracker/releases/latest) et [PopTracker](https://github.com/black-sliver/PopTracker/releases/latest).
2. Mettre le fichier du tracker dans le sous-répertoire /packs/ du répertoire d'installation de PopTracker.
3. Lancez PopTracker, et ouvrez le pack d'Aquaria.
4. Pour activer le tracking automatique, cliquez sur le symbole "AP" dans le haut de la fenêtre.
5. Entrez l'adresse du serveur Archipelago (le serveur auquel vous avez connecté le client), le nom de votre slot, et le mot de passe (si un mot de passe est nécessaire).

Le logiciel vous indiquera si une mise à jour du pack est disponible.
```

File: worlds/aquaria/docs/setup_fr.md
Comment (Cipocreep) [reply worlds/aquaria/docs/setup_fr.md:130]: Yeah, it's tioui is right, let's not lose sleep over this

Comment (benny-dreamly) [reply worlds/aquaria/docs/setup_fr.md:130]: 100%

**Review by benny-dreamly — APPROVED** [review]
approval for only the English part of the PR for the time being

**Review by benny-dreamly — APPROVED** [review]
French section of the PR looks good to me.

**Review by Cipocreep — APPROVED** [review]

**Review by NewSoupVi — APPROVED** [review]

**Pull request merged**
